web/collections.textile

```diff
@@ -45,19 +45,39 @@ numbers: Array[Int] = Array(1, 2, 3, 4, 5, 1, 2, 3, 4, 5)
 scala> numbers(3) = 10
 </pre>
 
+h2(#Seqs). Seqs
+
+Seqs preserve order, can contain duplicates, and are immutable (by default). Note that there are actually three different kinds of Seqs: scala.collection.Seq (trait), scala.collection.mutable.Seq, scala.collection.immutable.Seq. The latter is used by default. Note that List and other collections implement the Seq trait, so it's good practice to default to using it for immutable, ordered lists of things.
+
+<pre>
+scala> val numbers = Seq(1, 2, 3, 4, 5, 1, 2, 3, 4, 5)
+numbers: Seq[Int] = Seq(1, 2, 3, 4, 5, 1, 2, 3, 4, 5)
+
+scala> numbers(3) = 10
+<console>:9: error: value update is not a member of Seq[Int]
+              numbers(3) = 10
+</pre>
+
+
 h2(#Lists). Lists
 
-Lists preserve order, can contain duplicates, and are immutable.
+Lists preserve order, can contain duplicates, and are immutable. They are backed by a linked-list implementation, and enable cool pattern matching:
```

Comment:
Maybe mention that linked lists are popular in functional programming because they are a persistent data structure:
https://en.wikipedia.org/wiki/Persistent_data_structure